x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_trends.ts

```diff
@@ -23,18 +24,30 @@ export interface ScoreTrendDoc {
       failed_findings: number;
     }
   >;
+  score_by_benchmark_id: Record<
+    string,
+    Record<
+      string,
+      {
+        total_findings: number;
+        passed_findings: number;
+        failed_findings: number;
+      }
+    >
+  >;
```

Comment:
I'm suggesting to refactor the interface in order to avoid code duplication and using ovject instead of `Record`:

```typescript
interface FindingsDetails {
  total_findings: number;
  passed_findings: number;
  failed_findings: number;
}

interface ScoreByClusterId {
  [clusterId: string]: FindingsDetails;
}

interface ScoreByBenchmarkId {
  [benchmarkId: string]: {
    [key: string]: FindingsDetails;
  };
}

export interface ScoreTrendDoc {
  '@timestamp': string;
  total_findings: number;
  passed_findings: number;
  failed_findings: number;
  score_by_cluster_id: ScoreByClusterId;
  score_by_benchmark_id: ScoreByBenchmarkId;
}

```